ee/fleetd-chrome/src/background.ts

```diff
@@ -236,6 +237,12 @@ class NodeInvalidError extends Error {
   }
 }
 
+// We use a mutex to ensure that only one instance of main is running at a time.
+const mutexWithTimeout = withTimeout(new Mutex(), 60 * 1000) // 60 second timeout
```

Comment:
@getvictor Is there a reason you chose 60 seconds?

Reply:
No specific reason. Seemed a reasonable timeout, considering some tables like `geolocation` may be doing HTTP requests.

Reply:
Feels like good wiggle room, thanks for this fix!